general-superstaq/general_superstaq/check/all_.py

```diff
@@ -50,6 +61,13 @@ def run(*args: str, sphinx_paths: Optional[List[str]] = None) -> int:
         dest="force_all",
         help="'Hard force' ~ continue past (i.e. do not exit after) all failing checks.",
     )
+    parser.add_argument(
+        "--skip",
+        choices=CHECK_LIST,
+        nargs="*",
```

Comment:
```suggestion
        nargs="+",
        action="extend",
```

Reply:
`action="extend"` makes sense to me, but I think we want [`nargs="*"`](https://docs.python.org/3/library/argparse.html#nargs):
![image](https://github.com/Infleqtion/client-superstaq/assets/567092/5d4aba30-e2c1-4faf-9a46-21d48ec3f6b9)

Reply:
Oh: trying out `nargs="+"` works just fine though.  What am I not getting?

Reply:
Ah gotcha.  Thanks!